pipelines/skylab/smartseq2_single_nucleus/SmartSeq2SingleNucleus.wdl

```diff
@@ -0,0 +1,116 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/TrimAdapters.wdl" as TrimAdapters
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/StarAlign.wdl" as StarAlignFastq
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/Picard.wdl" as Picard
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/FeatureCounts.wdl" as CountAlignments
+import "https://raw.githubusercontent.com/broadinstitute/warp/snSS2_first_wdls/tasks/skylab/LoomUtils.wdl" as LoomUtils
+
+
+workflow SmartSeq2SingleNucleus {
+  meta {
+    description: "Process SmartSeq2 snRNA-Seq data, include reads alignment, QC metrics collection, and gene expression quantitication"
```

Comment:
```suggestion
    description: "Process SmartSeq2 snRNA-Seq data including read alignment, QC metrics collection, and gene expression quantification"
```

And maybe at the end of the line we want to say something like "over intronic and exonic regions" ?

Reply:
updated.